.github/workflows/ubuntu.yml

```diff
@@ -95,3 +95,11 @@ jobs:
       - name: Print installed versions
         if: always()
         run: .github/workflows/print_versions.sh
+  build-and-test-success:
+    name: Build & Test Result
```

Comment:
This name is already generic to handle checking for more than this OS.

Reply:
I guest one part I don't understand is where these aggregators will live. This one is in Ubuntu YAML.

Reply:
It looks like this: 
![image](https://github.com/OSGeo/grass/assets/27212526/e164cb0b-008b-496c-b34b-5aec8cd3e326)
![image](https://github.com/OSGeo/grass/assets/27212526/afa8f159-4a57-473d-bacb-b5f0d804846e)
![image](https://github.com/OSGeo/grass/assets/27212526/fae85ba6-f5ee-44b1-8e58-2b511ba6983b)

Reply:
The result looks promising. Let's try. The YAML files can be restructured if needed.